lib/_stream_writable.js

```diff
@@ -339,6 +337,8 @@ function onwrite(stream, er) {
     }
 
     if (sync) {
+      // TODO(trevnorris): Yank this, but first need to figure out why
```

Comment:
I will. Just don't have the brain cells at the moment to figure out why `process.nextTick()` returns `undefined` in the mentioned test.